trace/cloud-trace-demo-app-opentelemetry/README.md

```diff
@@ -0,0 +1,73 @@
+# cloud-trace-demo-app-opentelemetry
+
+Open this demo app in [Google Cloud Shell](https://cloud.google.com/shell/docs/). This includes necessary tools.
+
+
+[![Open Cloud Trace Demo APP in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=trace/cloud-trace-demo-app/README.md&amp;cloudshell_tutorial=trace/cloud-trace-demo-app/README.md)
+
+#### Demo Requirements
+If you are using Cloud Shell, skip to the next section.
+
+1. Install gcloud <https://cloud.google.com/sdk/install>
+2. Install kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl/>
+3. Install docker <https://docs.docker.com/install/>
+
+    
+#### Create a GKE cluster
+
+8. Enable Google Cloud and set up region and zone.
+
+    `gcloud init`
+9. Enable the GKE API & billing:
+
+    `gcloud services enable container.googleapis.com`
+10. Create a GKE cluster named "cloud-trace-demo", replace you GCP zone with
+your gcp-zone closest in proximity to you:
+
+     ```
+     gcloud container clusters create cloud-trace-demo\`
+     --num-nodes 1 \
+     --enable-basic-auth \
+     --issue-client-certificate \
+     --zone your-gcp-zone
+     ```
+12. Verify that you have access to the cluster:
+    
+    `kubectl get nodes`
+
+#### Deploy The Cloud Trace Demo App
+
+13. Build and tag the docker image for demo app:
+
+    `docker build -t gcr.io/${PROJECT_ID}/cloud-trace-demo:v1 .`
+14. Deploy resource to the cluster.
+
+    `kubectl apply -f deployment.yaml`
+15. Track the status of the deployment:
+
+    `kubectl get deployments`
+    
+    Deployment is complete, when all of the available deployments are ready. 
+16. Run the following command to see the pods the deployment created:
+    
+    `kubectl get pods`  
+
+#### Deploy The Cloud Trace Demo Service
+
+17. Create the cloud trace demo service
+    
+    `kubectl apply -f service.yaml`
+18. Get the services IP address by running the following command:
+    
+    `kubectl get services`
+19. Send curl request to the EXTERNAL_IP, replcae EXTERNAL_IP with the external ip found
+in the above command:
```

Comment:
Instead of "in the above command" it would be better to be more specific and say "in step 18" (or whatever number that step gets after the update from another comment).